tests/unittests/Foundation/NSFileHandleTestFile.txt

```diff
@@ -1 +0,0 @@
-The Quick Brown Fox.
```

Comment:
The file was being generated and then deleted by a few tests.  It probably was accidentally left at some point, but it doesn't seem to be needed.